Sources/Foundation/NSPathUtilities.swift

```diff
@@ -814,8 +818,8 @@ internal func _NSCreateTemporaryFile(_ filePath: String) throws -> (Int32, Strin
         close(fd)
         throw _NSErrorWithErrno(_errno, reading: false, path: pathResult)
     }
-#endif
     return (fd, pathResult)
+#endif
```

Comment:
this change seems to be impacting non-WASI `#if` branch?

Reply:
The `return (fd, pathResult)` line was shared among all cases, but it's now cloned into `os(Windows)` and `!os(Windows) && !os(WASI)` cases. So there is no semantic change.